tt_metal/impl/dispatch/kernels/cq_prefetch.cpp

```diff
@@ -149,6 +157,10 @@ void read_from_pcie(volatile tt_l1_ptr uint32_t *& prefetch_q_rd_ptr,
     if ((uint32_t)prefetch_q_rd_ptr == prefetch_q_end) {
         prefetch_q_rd_ptr = (volatile tt_l1_ptr uint32_t*)prefetch_q_base;
     }
+
+    // NOT_STALLED Persists. STALL_NEXT -> STALLED.
+    stall_state = static_cast<StallState>(stall_state << 1);
```

Comment:
given this relies on specific values of the enum, let's assign the enum values explicitly, eg, NOT_STALLED=0